packages/SketchMorph2-Tests.package/M2BackendTests.class/instance/brushSizeTestUsingUndo.testing..st

```diff
@@ -0,0 +1,26 @@
+testblocks
+brushSizeTestUsingUndo: anUndoBlock testing: aTestBlock
+	self backend
+		fillShape;
+		penColor: Color blue.
+	self drawAt: 10 asPoint.
+	self compareFormWith: #blueBackground.
+	self backend penColor: Color white.
+	self drawAt: 20 asPoint.
+	self compareFormWith: #whiteBackground.
+	self backend
+		setBrushAndEraserSizeTo: 50;
+		drawCircle;
+		penColor: Color black.
+	self drawAt: 50 asPoint.
+	self compareFormWith: #brushSizeTest.
+	self backend setBrushAndEraserSizeTo: 40.
+	self drawAt: 20 asPoint.
+	"TODO: Add assert that this actually changed something."
```

Comment:
Remove comment.

Reply:
Yes, but only after we found the time to create a comparison form for this case. This test would currently pass when the `drawAt:` call didn't change anything, ignoring everything `undo` could do wrong.